src/main/java/com/softserve/academy/spaced/repetition/logger/LogException.java

```diff
@@ -1,27 +0,0 @@
-package com.softserve.academy.spaced.repetition.logger;
-
-import org.aspectj.lang.JoinPoint;
-import org.aspectj.lang.annotation.AfterThrowing;
-import org.aspectj.lang.annotation.Aspect;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
-
-@Aspect
-@Component
-public class LogException {
```

Comment:
This class must stay, but use a different logger. Please read the description in the Jira case once again.